website/content/docs/enterprise/consistency.mdx

```diff
@@ -0,0 +1,212 @@
+standbys with Raft Integrated Storage, or when using performance replication,
+there are some sequences of operations that don't always yield read-after-write
+consistency.
+
+## Performance Standby Nodes
+
+When using Consul as a storage backend, every Vault node gets a consistent
+view of storage.  This is because the default Consul consistency model sends
+all requests to the leader node.
+
+When using the Integrated Storage backend without performance standbys, only
+a single Vault node (the active node) handles requests.  Requests sent to
+regular standbys are handled by forwarding them to the active node.
+
+When using the Integrated Storage backend with performance standbys, both the
+active node and performance standbys can handle requests.  If a performance standby
+handles a login request, or a request that generates a dynamic secret, the
+performance standby will issue an RPC to the active node to store the token
+and/or lease.  If the performance standby handles any other request that
+results in a storage write, it will forward that request to the active node
+in the same way a regular standby forwards all requests.
+
+With integrated storage all writes occur on the active node, which then issues
+RPCs to update the local storage on every other node.  Between when the active
+node writes the data to its local disk, and when those RPCs are handled on the
+other nodes to write the data to their local disks, those nodes present a stale
+view of the data.
+
+As a result, even if you're always talking to the same performance standby,
+you may not get read-after-write semantics.  The write gets sent to the active
+node, and if the subsequent read request occurs before the new data gets sent
+to the node handling the read request, the read request won't be able to take
+the write into account because the new data isn't present on that node yet.
+
+## Performance replication
+
+A similar phenomenon occurs when using performance replication.  One example
+of how this manifests is when using shared mounts.  If a KV secrets engine
+is mounted on the primary with `local=false`, it will appear on the secondary
+cluster as well.  The secondary cluster can handle requests to that mount,
+though as with performance standbys, write requests must be forwarded - in
+this case to the primary active node.  Once again, if a write request is followed
+by a read request, and the data initially written on the primary cluster active
+node hasn't yet been pushed out to the secondary cluster before the read request is
+processed on the secondary node handling the request, it will appear as though
+the write hasn't happened.  If the secondary cluster is using Integrated Storage,
+and the read request is being handled on a performance standby, the problem is
+exacerbated because it has to be sent first from the primary active node to the
+secondary active node, and then from there to the secondary performance standby.
```

Comment:
A call flow diagram ala https://www.researchgate.net/profile/Mayank-Raj/publication/228693373/figure/fig2/AS:302067169153025@1449029833162/A-Typical-Call-flow-Diagram-for-the-Proposed-Architecture.png would be worth 1000 words here.